pkgs/top-level/all-packages.nix

```diff
@@ -32701,6 +32701,8 @@ with pkgs;
 
   meerk40t-camera = callPackage ../applications/misc/meerk40t/camera.nix { };
 
+  mml = callPackage ../tools/text/mml { };
```

Comment:
Consider initializing this in `pkgs/by-name` instead

Reply:
Sorry if the question sounds stupid, but what is the difference between both?

Reply:
> Sorry if the question sounds stupid

It doesn't!

You can read more here https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#name-based-package-directories

Basically, by adding it as `pkgs/by-name/mm/mml/package.nix` you don't have to put it in top-level